### PR TITLE
Closes #1 Create initial index.html

### LINKED ISSUE #1: create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

**Comment by orestsyn** [comment]
add empty line at the end

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (AndriyKV) [inline index.html]: Please, add empty line.

**Comment by AndriiTur** [comment]
add empty line

**Comment by AndriyKV** [comment]
Please, add empty end line.

**Pull request merged**